javascript/javascript3/week2/fetchandwait.js

```diff
@@ -0,0 +1,31 @@
+//Fetching and waiting
+/*The 3 steps:
+1. Wait 3 seconds. 2. After 3 seconds fetch some data from any api you like 3.Log out the data from the api */
+
+//using promises and .then
+const astronautApi = 'http://api.open-notify.org/astros.json';
+const promise = new Promise((resolve,reject)=>{
+  setTimeout(()=>{
+    resolve();
```

Comment:
`Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)`
In this case you can call `resolve("got data :)")